app/components/TransferPointNode.tsx

```diff
@@ -88,4 +150,16 @@ const createStyles = (theme: MD3ThemeExtended) =>
       left: "100%",
       height: 20,
     },
+    leftComponent: {
+      position: "absolute",
+      top: "50%",
+      transform: [{ translateY: -22 }],
+      right: "115%",
+      display: "flex",
+      alignItems: "center",
+    },
+    leftComponentButton: {
+      margin: 0,
```

Comment:
Isn't margin 0 by default?

Reply:
It isn't :P but i refactored it a little bit now it should work better